Project/Assets/Editor/Annotations/AnnotationInteractionEditor.cs

```diff
@@ -14,20 +14,60 @@ public override void OnInspectorGUI()
     {
         annotation = (Annotation)target;
 
-        string[] options = new string[] { "Import Text File", "Write in Inspector" };
-        annotation.selected = EditorGUILayout.Popup("Import Method", annotation.selected, options);
+
+        string[] typeOptions = new string[] { "Summary Annotation", "Area Annotation" };
+        annotation.annotationType= EditorGUILayout.Popup("Annotation Type", annotation.annotationType, typeOptions);
+
+        if(annotation.annotationType == 0)
+        {
+            DisplaySetSummary();
+        }
+        else
+        {
+            ///Stuff for Area Annotations here
+        }
+
+        int before = annotation.importType;
+
+        string[] importOptions = new string[] { "No Full Annotation", "Import Text File", "Write in Inspector" };
+        annotation.importType = EditorGUILayout.Popup("Import Method", annotation.importType, importOptions);
+
+        //checks with user, then resets full annotation information
+        if (before != 0 && annotation.importType == 0)
+        {
+            if (EditorUtility.DisplayDialog("Reset", "WARNING: Switching back to 'No Full Annotation' will cause any curent information to be lost...",
+                "Continue", "Cancel"))
+            {
+                annotation.imagePath = "";
+                annotation.text = "";
+                annotation.textFilePath = "";
+                annotation.textFile = null;
+            }
+            else
+            {
+                annotation.importType = before;
+            }
+        }
 
         //Processing if importing a text file
-        if (annotation.selected == 0)
+        if (annotation.importType == 1)
```

Comment:
Also, could you extract these numbers into constants? So instead of `annotation.importType == 1`, you could have `annotation.importType == AREA_ANNOTATION` where `AREA_ANNOTATION = 1`? (Or whatever type corresponds to 1)